# HG changeset patch
# User Rik <rik@octave.org>
# Date 1356642912 28800
#      Thu Dec 27 13:15:12 2012 -0800
# Node ID 98f6abe01b5aaf2d867989babbdb7e4f7df2bf0f
# Parent  e77cd781eeef8430ed9ae617150a83c38ea64a52
configure.ac: Remove unnecessary double quotes in test expression.

* configure.ac: Remove unnecessary double quotes in test expression.

diff --git a/configure.ac b/configure.ac
--- a/configure.ac
+++ b/configure.ac
@@ -2347,17 +2347,17 @@ do
     *-msdosmsvc*)
       build_java=true
       JAVA_LIBS=-ladvapi32
       AC_DEFINE(HAVE_JAVA, 1,
         [Define to 1 if Java is available and is at least version 1.5])
       break
     ;;
     *-mingw*)
-      if test "$have_msvc" = "yes"; then
+      if test $have_msvc = yes; then
         build_java=true
         JAVA_LIBS=-ladvapi32
         AC_DEFINE(HAVE_JAVA, 1,
           [Define to 1 if Java is available and is at least version 1.5])
         break
       fi
     ;;
   esac
